get-resource.sh

```diff
@@ -36,7 +36,18 @@ if [ -s "/shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENA
     echo "$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum found, contents:"
     cat /shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum
 else
-    curl -g --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}"
+    CONNECT_TIMEOUT=120
+    MAX_ATTEMPTS=5
+
+    for i in $(seq ${MAX_ATTEMPTS}); do
+        if ! curl -g --insecure --compressed -L --connect-timeout ${CONNECT_TIMEOUT} -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}"; then
```

Comment:
do we want to redirect the error to null here ?

Reply:
Don't we want to ensure any error is visible in the container logs?

Reply:
I agree, the error must remain visible

Reply:
sounds good to me :)